docker/hosted-mode/docker-compose.yaml

```
@@ -16,7 +16,7 @@ services:
     volumes:
       - "../../spec_files:/opt/mojaloop-testing-toolkit/spec_files"
```

```suggestion
      - "../../spec_files:/opt/app/spec_files"
```

Note: please check for any other places where the directory will need to be updated